src/Pages/Login/login.css

```diff
@@ -69,6 +54,27 @@
   width: 300px;
   height: 300px;
   position: relative;
-  transform: translate(-2%,0%);
+  justify-content: center;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, 0);
   top: -20px;
 }
+
+.form-card-login {
+  position: relative;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%,-50%);
+  width: 20%;
+  min-width: 400px;
+  
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.26);
+  border-radius: 20px;
+  padding: 1.5rem;
+  background: white;
+  flex: none;
+
+  margin: 0;
+  justify-content: center;
+}
```

Comment:
That means to add an empty line at the end of the file. [Why?](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file)

![image](https://user-images.githubusercontent.com/18486562/80544544-1d54fd00-8966-11ea-8a0f-b1941a8e65c6.png)